debian/rules

```diff
@@ -241,6 +241,8 @@ override_dh_auto_install:
 		rm -r debian/tmp/$$p ;		\
 	done
 	touch debian/tmp/etc/machine-id
+	# This hack should be removed with #113
+	sed -i '/^After=/{;s, *plymouth-start[.]service *, ,;/^After= *$$/d;}' debian/tmp/usr/lib/systemd/system/systemd-ask-password-*
```

Comment:
FTR this creates an empty `After=` in `systemd-ask-password-console.path`, which is probably fine

Reply:
I do not see it.

```
cat debian/tmp/usr/lib/systemd/system/systemd-ask-password-console.path | grep After=
```

Gives nothing.

Reply:
This is what I saw when applying the command in the file:
```
$ sed '/^After=/{;s, *plymouth-start[.]service *, ,;/^After= *$$/d;}' usr/lib/ubuntu-core-initramfs/main/usr/lib/systemd/system/systemd-ask-password-console.path
#  SPDX-License-Identifier: LGPL-2.1-or-later
#
#  This file is part of systemd.
#
#  systemd is free software; you can redistribute it and/or modify it
#  under the terms of the GNU Lesser General Public License as published by
#  the Free Software Foundation; either version 2.1 of the License, or
#  (at your option) any later version.

[Unit]
Description=Dispatch Password Requests to Console Directory Watch
Documentation=man:systemd-ask-password-console.path(8)
DefaultDependencies=no
Conflicts=shutdown.target emergency.service
After= 
Before=paths.target shutdown.target cryptsetup.target
ConditionPathExists=!/run/plymouth/pid

[Path]
DirectoryNotEmpty=/run/systemd/ask-password
MakeDirectory=yes
```

Reply:
But the `$$` will be changed to `$` by `make`.

Reply:
Try running the same command through make.

Reply:
Oh, right, didn't notice that. Thanks!